quiz-app/css/question.css

```diff
@@ -0,0 +1,150 @@
+.question-container{
+    display: flex;
+    align-items: center;
+    justify-content:center;
+    height: 100vh;
+}
+ .quiz-box{
+    width: 550px;
+    background: var(--COLOR-WHITE);
+    border-radius: 0.5rem;
+}
+
+.quiz-box header{
+    height: 70px;
+    padding: 0 30px;
+    background: var(--COLOR-WHITE);
+    border-radius: 5px 5px 0 0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.quiz-box-title{
+    font-size: 20px;
+    font-weight: 600;
+}
+
+.quiz-box-timer{
+    color: var(--COLOR-TIMER-TEXT);
+    background: var(--COLOR-TIMER-BKGD);
+    border: 1px solid var(--COLOR-TIMER-BORDER);
+    height: 45px;
+    padding: 0 0.5rem;
+    border-radius: 0.5rem;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 145px;
+}
+
+.quiz-box-timer .time_left_txt{
+    font-weight: 400;
+    font-size: 17px;
+    user-select: none;
+}
+
+.quiz-box-timer .timer_sec{
+    font-size: 18px;
+    font-weight: 500;
+    height: 30px;
+    width: 45px;
+    color: var(--COLOR-WHITE);
+    border-radius: 5px;
+    line-height: 30px;
+    text-align: center;
```

Comment:
use text-center as helper class, in this way you don't have to repeat the code your self